apps/web/lib/orgMigration.ts

```diff
@@ -251,21 +264,30 @@ async function dbMoveTeamToOrg({ teamId, targetOrgId }: { teamId: number; target
     });
   }
 
-  if (team.parentId === targetOrgId) {
-    log.warn(`Team ${teamId} is already in org ${targetOrgId}`);
-    return team;
-  }
+  const teamMetadata = teamMetadataSchema.parse(team?.metadata);
+  const oldTeamSlug = teamMetadata?.migratedToOrgFrom?.teamSlug || team.slug;
 
-  await prisma.team.update({
+  const updatedTeam = await prisma.team.update({
     where: {
       id: teamId,
     },
     data: {
-      parentId: targetOrgId,
+      slug: targetOrg.teamSlug,
+      parentId: targetOrg.id,
+      metadata: {
+        ...teamMetadata,
+        migratedToOrgFrom: {
+          teamSlug: team.slug,
+          lastMigrationTime: new Date().toISOString(),
+        },
```

Comment:
Just saw this - how should we store this info in the new organization settings PR?